homeassistant/components/n26/__init__.py

```diff
@@ -17,14 +17,12 @@
 DEFAULT_SCAN_INTERVAL = timedelta(minutes=30)
 
 # define configuration parameters
-CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
This is not the proper way to do this. PLATFOR_SCHEMA is only for platforms - but this is a component.

Change this to use CONFIG_SCHEMA again with

```suggestion
    DOMAIN: vol.All(cv.ensure_list, vol.Schema({
```